Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -390,7 +390,7 @@ export default class OrderForm extends React.Component<IOrderFormProps, IOrderFo
     }
 
     private _checkUcChart = (chart: string) => {
-        return (chart === "L" || chart === "l" || chart === "3");
+        return (chart === "L" || chart === "l" || chart === "3" || chart === "S" || chart === "s" || chart === "M" || chart === "m");
```

Comment:
@jSylvestre fine but you can use ToUpper() and then you'd only have to do half as many comparisons